pkg/reconciler/labeler/v1/accessors.go

```diff
@@ -71,9 +71,9 @@ func (r *Revision) list(ns, name string) ([]kmeta.Accessor, error) {
 		return nil, err
 	}
 	// Need a copy to change types in Go
-	kl := make([]kmeta.Accessor, 0, len(rl))
-	for _, r := range rl {
-		kl = append(kl, r)
+	kl := make([]kmeta.Accessor, len(rl))
+	for i, r := range rl {
+		kl[i] = r
```

Comment:
Why this change? I looked that up recently and the consensus was that there's no tangible perf difference between the two.

Reply:
I thought it was clearer that this is a straight copy rather than building a new array. Maybe even better would be:
```
for i range rl {
    kl[i] = rl[i]
}
```

Reply:
If it's a straight copy why not just `copy(kl, rl)`? (I think that's also very slightly faster)

Reply:
oh wait, because they're different types ☺️, I read the comment now!